### ISSUE TITLE
world breakage

### ISSUE THREAD

**Issue by TheNimbleNinja** [DoctorFTB/FluidCows#9]
when you spawn a cow with the rocket_fuel tag  the model breaks and causes the world to not be playable. you can not see a thing and cant log out of the world and will be forced to close the game.

**Comment by DoctorFTB** [comment]
From what mod "rocket_fuel"?

**Comment by DoctorFTB** [comment]
@TheNimbleNinja

**Comment by DoctorFTB** [comment]
@TheNimbleNinja

**Comment by EpicEricMao** [comment]
It happens the same here. After I looked at the "milk cow", the playscreen seems to be broken. Though others still works as well, it does'nt shows HUD or renders correctly. After I removed this mod, it was fixed. BTW, another mod Dragon Mount has akin problem.

And I'm Chinese, so my English might be some strange, hope it can be fixed soon. THX.

**Comment by DoctorFTB** [comment]
@EpicEricMao I download mod "Dragon Mounts 2" and I not found any fluids in mod.
I need more info..

**Comment by EpicEricMao** [comment]
Well, I meant DragonMount has a similar question, there are nothing connected. I didnt use DragonMount then. And after I recieve your reply, I download FluidCow again. Strangely, when I created another world, there was no bug (I was using the same mods), then I went back to the original world, the same bug appers again.

Details: 
This time, at first, when I open my inventory and look for "Fluid Cow (Milk)", it shows as a cow-shaped black thing. After I summon it and look at it, the picture is so strange, like "has a furnace on its body" or "has some structure of some items (beetroot seed, carrot, etc.)". last time it shows like lots of horizontal stripes. Of course after I removed the mod everything get back to normal.

At last, thank you for forgiving my bad English (-_-T).
